api/tests/unit/application/usecases/checkOrganizationIsScoAndManagingStudent_test.js

```diff
@@ -0,0 +1,50 @@
+const { expect, sinon, domainBuilder } = require('../../../test-helper');
+const useCase = require('../../../../lib/application/usecases/checkOrganizationIsScoAndManagingStudent');
+const organizationRepository = require('../../../../lib/infrastructure/repositories/organization-repository');
+
+describe('Unit | Application | Use Case | checkOrganizationIsScoAndManagingStudent', function () {
+  context('When organization is SCO managing students', function () {
+    it('should return true', async function () {
+      // given
+      sinon.stub(organizationRepository, 'get');
```

Comment:
```suggestion
const organizationRepository = { get: sinon.stub() }
```

Pour avoir ça en moins à faire lors de la migration ESM.

Reply:
Je peux faire un commit à part pour modifier ce usecase pour préparer la migration ESM effectivement